todo_app/app.py

```diff
@@ -1,11 +1,65 @@
 from flask import Flask
+from flask import render_template, redirect, url_for, flash
+from flask import request
 
 from todo_app.flask_config import Config
+import todo_app.data.session_items as session_items
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    '''Main page'''
+    return render_template('index.html', items=session_items.get_items())
+
+@app.route('/changeItem', methods=['POST'])
+def changeItem():
+    '''Can create, edit, and delete an item'''
+    reqAction = request.form.get('changeItem')
+
+    if reqAction == 'Add Item':
+        itemTitle = request.form.get('itemTitle')
+        if itemTitle == "":
+            flash('ERROR: No title specified for new item')
+        else:
+            session_items.add_item(itemTitle)
+
+    if reqAction == 'Delete Item':
+        itemID = request.form.get('itemID')
+        if itemID == "":
+            flash('ERROR: No ID specified for item to be deleted')
+        else:
+            session_items.delete_item(itemID)
+
+    if reqAction == 'Get Item':
+        itemID = request.form.get('itemID')
+        if itemID == "":
+            flash('ERROR: No ID specified for item to be retrieved')
+        else:
+            item = session_items.get_item(itemID)
+            itemStart = item['status'] == 'Started'
+            itemTitle = item['title']
+            return render_template('index.html', items=session_items.get_items(),
+                                    setID=itemID, setStart=itemStart, setTitle=itemTitle)
```

Comment:
nitpick: in Python the variable and function naming convention is the same: lower case separated by "_". Here and elsewhere it would be `item_id`, `item_start`, etc.. https://stackoverflow.com/questions/159720/what-is-the-naming-convention-in-python-for-variable-and-function

The hand waving reason for it is that it makes python read more like natural language, which it aspires to be more like compared to many other languages, where you'd often see camelCase and PascalCase as conventions